cdap-ui/app/cdap/components/FormInputs/Password/index.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import React, { useState } from 'react';
+import TextField from '@material-ui/core/TextField';
+import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
+import ThemeWrapper from 'components/ThemeWrapper';
+import PropTypes from 'prop-types';
+import InputAdornment from '@material-ui/core/InputAdornment';
+import IconButton from '@material-ui/core/IconButton';
+import Visibility from '@material-ui/icons/Visibility';
+import VisibilityOff from '@material-ui/icons/VisibilityOff';
+
+
+const styles = (theme) => {
```

Comment:
This files breaks the build. Mainly because of tslint formatting. Might not be required but I would suggest have tslint extension in vscode installed and enabled with format on save option. That would help us prevent this error.

Reply:
We may need to revisit the weback config for prod build to actually include tslint checks. :sigh: